_posts/2021-09-30-building-opensearch-1.1-distributions.markdown

```diff
@@ -0,0 +1,50 @@
+
+In parallel, a snapshot build builds and pushes `-SNAPSHOT` maven artifacts to [aws.oss.sonatype.org](https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/). 
+
+Unlike snapshots, release artifacts are signed. The signing step takes the manifest file created from the build step, and signs all its component artifacts using a tool called `opensearch-signer-client` (in progress of being open sourced). These release candidate artifacts are staged, tested, and final releases are promoted to [Maven Central](https://repo1.maven.org/maven2/org/opensearch/) manually.
+
+A [test workflow](https://github.com/opensearch-project/opensearch-build/blob/0ac7e2bfcf6adbbd49d7a2b3fff59bb9eea28a61/bundle-workflow/README.md#test-the-bundle) is executed for each build, and consisting of integration, backwards compatibility and performance tests. The [test orchestrator pipeline](https://github.com/opensearch-project/opensearch-build/tree/main/bundle-workflow/jenkins_workflow/test) and how to run tests locally will be a topic for a future blog post.
+
+Here's a quick cheat-sheet for reproducing the build process for a complete distribution of OpenSearch. You will need additional tools, including `maven` and `cmake`, to compile some of the components.
+
+```sh
+git clone https://github.com/opensearch-project/opensearch-build
+cd opensearch-build
+./bundle-workflow/build.sh manifests/opensearch-1.1.0.yml
+./bundle-workflow/assemble.sh manifests/opensearch-1.1.0.yml
+```
+
+This system will be reused for OpenSearch Dashboards in 1.2, see [opensearch-build#158](https://github.com/opensearch-project/opensearch-build/issues/158).
+
+The signing process continues, for now, to rely on an existing signing system and all our jobs run from a private Jenkins setup. This is entirely for historical reasons. The OpenSearch team at AWS relies on a significant amount of costly hardware procured and managed by AWS (e.g. I use a c5.18xlarge instance for development). We are in the process of making the Jenkins instance public, open-sourcing its CDK setup, and open-sourcing all remaining tools.
+
+The build process described above runs parallel to development of individual components. It's always challenging to code against moving dependencies, and we are starting to think about the long term future of these systems. We'd like to untangle the monolith, enable plugins to declare a compatibility matrix, and much more. In the meantime, we welcome anyone who would like to brush up their Python skills by contributing to [the opensearch-build scripts](https://github.com/opensearch-project/opensearch-build), and could use some help bringing OpenSearch and OpenSearch Dashboards to various OS-specific distribution mechanisms similar to [this community port to FreeBSD](https://github.com/opensearch-project/opensearch-build/issues/101).
```

Comment:
In the future, I'd like to avoid 'we' more - but I think since you defined it in the preceding paragraph, I'm fine with it.